app/Http/Controllers/SubGroupController.php

```
@@ -0,0 +1,99 @@
+        ]);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     */
+    public function update(SubGroupStoreRequest $request, SubGroup $subGroup)
+    {
+        $validatedData = $request->validated();
+        DB::transaction(function () use ($validatedData, $subGroup) {
+            $subGroup->update($validatedData);
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'update'. (undefined)

(StaticAccess)